micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/ExecutorServiceMetrics.java

```diff
@@ -402,6 +402,16 @@ private void monitor(MeterRegistry registry, ForkJoinPool fj) {
             .description(
                     "An estimate of the number of worker threads that are not blocked waiting to join tasks or for other managed synchronization threads")
             .register(registry);
+
+        Gauge.builder(metricPrefix + "executor.parallelism", fj, ForkJoinPool::getParallelism)
+            .tags(tags)
+            .description("The targeted parallelism level of this pool")
+            .register(registry);
+
+        Gauge.builder(metricPrefix + "executor.pool.size", fj, ForkJoinPool::getPoolSize)
+            .tags(tags)
+            .description("The number of worker threads that have started but not yet terminated")
```

Comment:
If we're using the same metric name, we should keep the description the same as the one for ThreadPoolExecutor to avoid issues with backends that don't accept different descriptions (see e.g. https://github.com/micrometer-metrics/micrometer/pull/3201)

> The current number of threads in the pool

Reply:
Wow, that's unexpected. FIxed